docs/EnvironmentVariables.md

```diff
@@ -1,7 +1,10 @@
 # Environment variables
 
-The PSGI server (`zonemaster_backend_rpcapi.psgi`) support the following
-environment variables.
+* `ZONEMASTER_BACKEND_CONFIG_FILE`: Set a custom path for the backend
+  configuration file.
+
+The following environement variables are only supported by the PSGI server
```

Comment:
In other places we're referring to "the RPCAPI daemon" or just "the RPCAPI". Maybe use that here as well?

Reply:
I agree, but instead I suggest:

```
## Variables used by RPCAPI daemon only
```

Reply:
done